client/ios/Callout/Info.plist

```diff
@@ -37,8 +37,14 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
```

Comment:
it wasn't clear which ones of these were required so I added all of them. We're supporting background location updates in the shell, now.